cdap-app-fabric/src/main/java/org/apache/twill/yarn/YarnTwillRunnerService.java

```diff
@@ -322,7 +321,6 @@ public synchronized TwillController lookup(String applicationName, final RunId r
   }
 
   private void startUp() throws Exception {
-    yarnAppClient.startAndWait();
```

Comment:
We made it no longer a Service, since we don't need to do anything in startup of it anymore.